File: 2388/2388.cs
public class Solution {
    public int ChangeNullValuesInATableToTheP(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(ChangeNullValuesInATableToTheP(root.left), ChangeNullValuesInATableToTheP(root.right));
    }
}

File: 2388/2388.go
func changeNullValuesInATableToTheP(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := changeNullValuesInATableToTheP(root.Left)
	r := changeNullValuesInATableToTheP(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}